openstack/compute/v2/flavors/requests.go

```diff
@@ -200,3 +200,14 @@ func IDFromName(client *gophercloud.ServiceClient, name string) (string, error)
 		return "", err
 	}
 }
+
+// ExtraSpecs requests all the extra-specs for the given flavor ID.
+func ListExtraSpecs(client *gophercloud.ServiceClient, flavorID string) (r ListExtraSpecsResult) {
+	_, r.Err = client.Get(extraSpecsListURL(client, flavorID), &r.Body, nil)
+	return
+}
+
+func GetExtraSpec(client *gophercloud.ServiceClient, flavorID string, key string) (r GetExtraSpecsResult) {
```

Comment:
Thanks for changing the function name to singular. I think the result type also be singular (It would have to be changed in other places, too) since the "internal" type is `extraSpecResult`?